### DISCUSSION TITLE
How to control the data provider not refresh when load the page

### DISCUSSION THREAD

**Discussion by touchsky2018** [Budibase/budibase#7557] [Help]
Is there any setting to control the data provider not refresh when load the page?

**Comment by aptkingston** [comment]
Hi @touchsky2018. I'm not sure what you mean. All data providers will refresh when the page loads.

Are you looking for a way to automatically refresh data providers on an interval?

They can be manually refreshed using the "refresh data provider" button action, but we don't have a way to automatically refresh them at the moment.
